bqetl_project.yaml

```diff
@@ -35,6 +35,217 @@ monitoring:
     - baseline_v1
     telemetry_stable:
     - main_v4
+    focus_ios_derived:
+    - active_users_aggregates_deletion_request_v1
+    - active_users_aggregates_v1
```

Comment:
The v1 and v2 should be deprecated and this, not need BigConfig, in case is relevant.